src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -242,7 +242,11 @@ export function transitionSortConfig(
         nextSortConfig.type === "order" &&
         (prevSortConfig.order !== nextSortConfig.order))
         ||
-        !_.isEqual(prevSortConfig, nextSortConfig)) {
+        !_.isEqual(prevSortConfig, nextSortConfig)
+    ) {
+        if (nextSortConfig.type === "cluster") {
+            oncoprint.removeAllExpansionTracksInGroup(nextSortConfig.track_group_index);
```

Comment:
And indeed, this is wrong too: I should remove these expansion tracks in the React state, not just in the Oncoprint model.